inference-engine/src/vpu/common/include/vpu/private_plugin_config.hpp

```diff
@@ -128,6 +128,15 @@ DECLARE_VPU_CONFIG(MYRIAD_FORCE_PURE_TENSOR_ITERATOR);
 // Myriad plugin options
 //
 
+/**
+ * @brief Used to prevent IE from booting MX VPU.
+ * This is a plugin scope option and must be used with the plugin's SetConfig method
+ * The only possible values are:
+ *     CONFIG_VALUE(YES)
+ *     CONFIG_VALUE(NO) (default value)
+ */
+DECLARE_VPU_CONFIG(MYRIAD_DISABLE_MX_BOOT);
```

Comment:
As for me, it is better to add "MYRIAD_ENABLE_MX_BOOT" and make "Yes" a default option.
This way is kept for many other myriad config options

Reply:
Done, thanks